lib/tlIO/FFmpegReadVideo.cpp

```diff
@@ -544,7 +559,7 @@ namespace tl
                     _avStream,
                     av_rescale_q(
                         time.value() - _timeRange.start_time().value(),
-                        swap(_avFormatContext->streams[_avStream]->r_frame_rate),
+                        swap(_avFormatContext->streams[_avStream]->avg_frame_rate),
```

Comment:
I don't think this takes into account the case where ```avg_frame_rate``` could be invalid?

Reply:
You are right.  I thought of that myself too.